Header header

uint8  station_num
uint16  switch_num
uint8  case_num